src/controllers/DocumentController.ts

```diff
@@ -55,7 +55,8 @@ export class DocumentController {
         let DOWNLOAD_URL: string | null = null;
 
         const documentResponse = await this.getCurrentDocumentState();
-        currentDocument = documentResponse.data ? JSON.parse(documentResponse.data) : null;
+        console.log(documentResponse, 'DMDMDMDMDMDMDMMDM');
```

Comment:
Should be removed

Reply:
Done 9cd4e66bdce0d3c165f978990ca0f51489e474bf